mslib/mscolab/server.py

```diff
@@ -259,7 +259,7 @@ def user_register_handler():
     try:
         if result["success"]:
             status_code = 201
-            if mscolab_settings.USER_VERIFICATION:
+            if mscolab_settings.MAIL_ENABLED:
                 status_code = 204
             token = generate_confirmation_token(email)
```

Comment:
this indenting looks wrong, likly we need only that token until sendmail when MAIL_ENABLED is True